portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -27,20 +31,17 @@
 /** Servlet that returns some example content. TODO: modify this file to handle comments data */
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
-  private final ArrayList<String> commentsList = new ArrayList();
-
-  @Override
-  public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
-    response.setContentType("application/json");
-    String json = new Gson().toJson(commentsList);
-    response.getWriter().println(json);
-  }
 
   @Override
   public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
       // Get input from the form.
-      String txt = getParameter(request, "text-input","");
-      commentsList.add(txt);
+      String comment = getParameter(request, "text-input","");
+      
+      // Store comments as entities in Datastore
+      Entity commentEntity = new Entity("Comment");
```

Comment:
Please create constant string value for this entity name, use that variable throughout your app.

Reply:
Yes, Yi-Tao also pointed it out, thank you for the information!